internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -603,6 +618,8 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) ResponseBody(respHeaders
 	}
 	headerMutation = &extprocv3.HeaderMutation{}
 	setContentLength(headerMutation, mut.Body)
+	o.tokenMetrics.UpdateTokenMetrics(o.backendName, o.modelName, tokenUsage.OutputTokens, tokenUsage.InputTokens, tokenUsage.TotalTokens)
```

Comment:
As mentioned in the previous review, these metrics are already recorded by the calling processor in its ProcessResponseBody method, once the translator returns.

It looks like the values will be incremened twice, which is not correct?

This is why it is important to have the tests I mention above, to make sure we're only recording metrics when needed, and not doing it N times by mistake.

This also applies to the other translators.